server/controller/order/postOrder.js

```diff
@@ -0,0 +1,23 @@
+const { insertOrder } = require('../../database/queries/order/addOrder');
+const { insertTuserOrder } = require('../../database/queries/captain/insertTuserOrder');
+const { insertItem } = require('../../database/queries/item/insertItem');
+
+exports.postOrder = (req, res) => {
+  console.log(req.body);
+  const {
+    address, items, phone, customerName, placeId, captainId,
+  } = req.body;
+  let orderId;
+  insertOrder(placeId, address, phone, customerName)
+    .then(({ rows }) => {
+      orderId = rows[0].pk_i_id;
+      return insertTuserOrder(captainId, orderId);
+    })
+    .then(() => insertItem(orderId, items))
+    .then(() => {
+      res.status(201).send({ result: 'the add order is done' });
```

Comment:
@engshorouq You could return the inserted ID here, that might be helpful for the frontend.

Reply:
I Know we need all data will insert it not just the id to display it after adding it in the table of orders